vim_binding.js

```diff
@@ -119,6 +119,30 @@ define([
         }
       }
   };
+
+  // soft-failing shortcut helper - works around the hard-failing shortcut
+  // manager in IPython
+  var add_shortcuts = function add_shortcuts(manager_name, data){
```

Comment:
> Named function expression

> If you want to refer to the current function inside the function body, you need to create a named function expression. This name is then local only to the function body (scope). This also avoids using the non-standard arguments.callee property.

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function

Wow. Did it changed sometime? I thought a named function expression would add the name into the global namescope thus I forced myself to use non named function expressions even it makes things difficult to debug...

Anyway thanks to aware me :-) Then this part is LGTM.